### PR TITLE
add appendix Feedback on this Book

### PR DESCRIPTION
**Pull request by KatherineCox**
One easy option for linking out to our feedback mechanism(s) - added a chapter to the appendix.

### CODE REVIEW

**Comment by KatherineCox** [comment]
Hmmm, getting a 404 when I try looking at the auto-generated preview.

Here's the rendered output from my machine

Screen Shot 2021-12-07 at 1 12 49 PM

File: 13-feedback.Rmd [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# Feedback on this Book
```
Comment (avahoffman) [inline 13-feedback.Rmd:1]: What do you think about making it a little more active? E.g., "Give us Feedback" ?

File: 13-feedback.Rmd [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+
+Thank you for your interest in this book! There are a few ways you can suggest improvements:
+
+<br>
+<!-- The capital letter above alters the formatting for the numbered points below -->
+
+1. Fill out this [Google form](https://docs.google.com/forms/d/e/1FAIpQLSeXD_1E4iUL005OXTLdt22XGW1q_W-6ThMsDmb7cotiCQu-2w/viewform?usp=sf_link).
```
Comment (avahoffman) [inline 13-feedback.Rmd:8]: Is this owned by the DaSL account we were talking about?

File: 13-feedback.Rmd [hunk 3]
```diff
@@ -0,0 +1,10 @@
...
+1. If you have a GitHub account, you can [raise an issue](https://github.com/jhudsl/AnVIL_Book_Getting_Started/issues) in our repository.
```
Comment (avahoffman) [inline 13-feedback.Rmd:9]: Maybe we could make this and/or number 3 the "preferred" method? At least until we get all the zapier stuff set up?

**Review by avahoffman — APPROVED** [review]
Looks good, assuming the render works!

File: 13-feedback.Rmd (hunk 2 above)
Comment (KatherineCox) [reply 13-feedback.Rmd:8]: Yes, I set up `DaSLZapier` Google, Zapier, GitHub accounts.  I'll share the info with you and others in the next couple days, just getting things working first.  This form should be good to go, though, unless we want to make edits to the questions.

Comment (avahoffman) [reply 13-feedback.Rmd:8]: great!

**Pull request merged**